lib/msal-common/src/scripts/metadata.js

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+const axios = require("axios");
```

Comment:
Axios has had a string of vulnerabilities lately, any alternatives we can use instead?

Reply:
I have replaced `axios` here with `https`